react/src/reactTextMask.js

```diff
@@ -1,41 +1,26 @@
-import React, {PropTypes} from 'react'
-import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
-
-export const MaskedInput = React.createClass({
-  propTypes: {
-    mask: PropTypes.oneOfType([
-      PropTypes.array,
-      PropTypes.func,
-      PropTypes.bool,
-      PropTypes.shape({
-        mask: PropTypes.oneOfType([PropTypes.array, PropTypes.func]),
-        pipe: PropTypes.func
-      })
-    ]).isRequired,
-    guide: PropTypes.bool,
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    pipe: PropTypes.func,
-    placeholderChar: PropTypes.string,
-    keepCharPositions: PropTypes.bool,
-    showMask: PropTypes.bool
-  },
-
-  createTextMaskInputElement,
```

Comment:
Because you can't just pass the value into a component class. You would be adding it to the instance, like "this.createttextmask = createTextMask" and then calling it on the instance. It's pointless. There was not a reason it needed to be there in the first place. It could have always been called directly